x-pack/plugins/security_solution/public/common/components/url_state/use_url_state.tsx

```diff
@@ -78,13 +96,15 @@ export const useUrlStateHooks = ({
             getParamFromQueryString(getQueryStringFromLocation(mySearch), urlKey) ??
             newUrlStateString;
           if (isInitializing || !deepEqual(updatedUrlStateString, newUrlStateString)) {
-            urlStateToUpdate = [
-              ...urlStateToUpdate,
-              {
-                urlKey,
-                newUrlStateString: updatedUrlStateString,
-              },
-            ];
+            if (updateTimelineAtinitialization(urlKey, newUrlStateString, urlState)) {
```

Comment:
nit: can't this be combined with the if above this?
```
          if (
            isInitializing ||
            (!deepEqual(updatedUrlStateString, newUrlStateString) &&
              updateTimelineAtinitialization(urlKey, newUrlStateString, urlState))
          ) {
```

Reply:
I do not think so because `isInitializing` will take over my function `updateTimelineAtinitialization`